packages/@uppy/dashboard/src/components/FileItem/Buttons/index.js

```diff
@@ -112,7 +115,12 @@ module.exports = function Buttons (props) {
         onClick={() => toggleFileCard(file.id)}
       />
       {showLinkToFileUploadResult && file.uploadURL ? (
-        <CopyLinkButton i18n={i18n} />
```

Comment:
Props for `file` were missing, so it was not possible to copy the link, //cc @goto-bus-stop just for reference

Reply:
whoooops thanks for fixing